### DISCUSSION TITLE
İngilizce ve türkçe kaynakları ayırsak nasıl olur ?

### DISCUSSION THREAD

**Discussion by YusufBilgin** [sdtrdev/source-pocket#143] [Ideas]
Linkler gene aynı sayfada olsa ama ufak bi filtre olsa başta ve türkçe ile ingilizce linkler arasında geçiş yapılsa sizce nasıl olur? 
Böylece sayfa daha düzenli olur ve yabancılarda source pocketi daha rahat kullanabilir.
Bu konu hakkında fikirleninizi merak ediyorum.

**Comment by omerayyildiz** [comment]
Güzel olur bence. Bunu daha önce düşünmüştüm. Linklerin başına [TR] ya da [EN] eklesek olur gibi. 🤔

**Comment by damla** [comment]
Ne gibi bir filtre düşünmüştün @YusufBilgin ? Ömer'in bahsettiği gibi mi yoksa farklı bir yöntem mi? Biliyorsun docusaurus kullandığımız için çok fazla özelleştirme yapamıyoruz, yapabileceklerimiz title eklemek veya keywordlerle yönetmek ile sınırlı 🤔

**Reply by YusufBilgin** [reply]
Aslında ben şöyle bir switch tarzı bişey düşünmüştüm ama evet docusaurus ile kullanımı mantıklı olmaz.

**Comment by mrabdullahsahin** [comment]
Aslında docusaurus çok dillilik özelliğini destekliyor. Çok dilli yapmak işimize yarar mı tam emin değilim ama bunu bir değerlendirebiliriz. Çok dillilik ile Türkçe için şu dosyaya bak İngilizce için şu dosyaya bak gibi bir durum olabilir eğer mümkünse, tam manasıyla incelemedim.
- [Bu v1 için ama v2 içinde vardır.](https://v1.docusaurus.io/docs/en/translation)
- [Multiple languages translation in Docusaurus](https://ledinhcuong99.medium.com/multiple-languages-translation-in-docusaurus-24f1ecfe526)

**Reply by damla** [reply]
Eğer iki dil için ayrı sayfa olarak ayarlanabiliyorsa mükemmel bir çözüm olabilir bizim için, teşekkürler @mrabdullahsahin 🥳

Mevcut süreçte bu feature ile ilgilenebilecek biri varsa bu tartışma veya Discord üzerinden bana ulaşabilirse sevinirim. Kimse çıkmaz ise ileriye yönelik planlara eklenmesini sağlarım.

**Reply by omerayyildiz** [reply]
Vay anasını sayın seyirciler.

**Reply by YusufBilgin** [reply]
0_0 ben belki 21 gün sonra ilgilenebilirim.

**Comment by damla** [comment]
#189
